src/nat/front_ends/fastapi/response_helpers.py

```
@@ -33,7 +36,9 @@ async def generate_streaming_response_as_str(payload: typing.Any,
                                              streaming: bool,
                                              step_adaptor: StepAdaptor = StepAdaptor(StepAdaptorConfig()),
                                              result_type: type | None = None,
-                                             output_type: type | None = None) -> AsyncGenerator[str]:
+                                             output_type: type | None = None,
+                                             send_done: bool = False,
```

Ruff (0.14.13)

Unused function argument: `send_done`

(ARG001)